src/AppBundle/Entity/Repositories/CategoryRepository.php

```diff
@@ -16,4 +19,13 @@ public function getWithJobs()
             ->getQuery()
             ->getResult();
     }
+
+    /**
+     * @param $slug
+     * @return null|object
+     */
+    public function findBySlug($slug)
```

Comment:
read about magic methods: https://symfony.com/doc/3.4/doctrine.html#fetching-objects-from-the-database
see example: `findByPrice`